src/main/java/com/futurcollars/lesson12/zad1/NameList.java

```diff
@@ -0,0 +1,29 @@
+package com.futurcollars.lesson12.zad1;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NameList {
+
+    NameList() {
+    }
+
+    List<String> name = Arrays.asList("Anna", "Katarzyna", "Tymon", "Tadeusz", "Anna", "Karolina", "Dorota", "Piotr");
+
+    public List<String> writeListInReverseOrder() {
+        Collections.reverse(name);
+        return name;
+    }
```

Comment:
Collections.reverse modyfikuje listę, więc jej użycie wpływa w tym przypadku na inne metody. Jeśli będziesz działąć na liście dostarczonej jako argument to nie będzie tego problemu :)